shared/src/domain-logic/opportunityFields.ts

```diff
@@ -1,6 +1,7 @@
 import { LookupLegalStructureById } from "../legalStructure";
 import { ProfileData } from "../profileData";
 import { determineForeignBusinessType } from "./businessPersonaHelpers";
+import { nexusLocationInNewJersey } from "./nexusLocationInNewJersey";
```

Comment:
[pebble] Let's update the import so it's not relative.

Reply:
we don't have relative imports set up for the shared directory